mirascope/rag/chunkers.py

```diff
@@ -0,0 +1,31 @@
+"""Chunkers for the RAG module."""
+import uuid
+from typing import Callable, Concatenate
+
+from pydantic import BaseModel
+
+from .types import Document
+
+
+def base_chunk_fn(
+    text: str,
+    chunk_size: int = 1000,
+    chunk_overlap: int = 0,
+) -> list[Document]:
+    """Chunk the text into chunks of size `chunk_size`."""
```

Comment:
also mention "and overlap `chunk_overlap`"

Reply:
this is removed now